src/moustachu.nim

```diff
@@ -70,7 +71,7 @@ proc parallelReplace(str: string,
   for sub in substitutions:
     result = result.replace(sub[0], sub[1])
 
-proc render(tmplate: string, contextStack: seq[Context]): string =
+proc render(tmplate: string, contextStack: seq[Context], pwd="."): string =
```

Comment:
Let's change `pwd` to `partials_dir` since the directory containing the partials files is not necessarily the present working directory.